tests/test_environment_indexentries.py

```diff
@@ -126,3 +126,21 @@ def test_create_index_by_key(app):
     assert index[0] == ('D', [('docutils', [[('main', '#term-docutils')], [], None])])
     assert index[1] == ('P', [('Python', [[('main', '#term-python')], [], None])])
     assert index[2] == ('ス', [('スフィンクス', [[('main', '#term-0')], [], 'ス'])])
+
+
+@pytest.mark.sphinx('dummy', freshenv=True)
+def test_create_index_with_name(app):
+    text = (".. index:: single: docutils\n   :name: ref1\n"
+            ".. index:: see: Python; interpreter\n   :name: ref2\n")
```

Comment:
Could you separate these text by each `\n` code? It make this text a bit readable.

Reply:
Done